biolink-model.yaml

```diff
@@ -858,6 +860,16 @@ slots:
 
   ## Biology
 
+  has phenotypic feature:
```

Comment:
How is this different from linking a `biological entity` to a `phenotypic feature` with `has phenotype` predicate?

Adding `has_*` as a property on nodes will lead to a proliferation of node properties that ought to be represented as edges.

Reply:
@cmungall has previously pointed out that there are more than one ways to represent knowledge, depending on specific  use cases.  

For sure, the 'has  phenotype' predicate is indicated a `biolink:Association ` to  link an  entity  (i.e. organism) to its phenotypic  profile,  where  the  `phenotypic  feature`  is not intrinsic to the entity concept itself,  i.e.  is in a more  conditional relationship.  

That said, in other situations, where a `phenotypic feature` is a formal descriptive part (a "defining characteristic") of the entity itself then connecting it directly to the object as a `node property` may make more sense  (i.e. so the `phenotypic  feature` travels along with the  concept).   A  couple of examples of this could include the following:

- the core 'invariant' characteristics of a disease, the specific `phenotypic feature`(s) which define it
- specific characteristics of instances of `biolink:Behaviour` 

In a sense, `has phenotypic feature` node property is a reified 'tight' relationship  of `phenotypic feature `to an entity.

Reply:
OK, @deepakunni3, after all the debates about best practices for node properties versus edge predicates, I have removed the `has phenotypic feature` node property and propagated the consequences (felt a need to add a new 'behavior' related association, to capture the spirit of my second example above...)